lib/git/commands.js

```diff
@@ -118,6 +118,30 @@ exports.listAllFiles = function(cwd, cb) {
   });
 };
 
+exports.listAdditionalPropertyFiles = function(modifiedRecords, cwd, cb) {
+
+  function is_properties_not_modified(modifiedRecords, file) {
+    return file.substr(file.lastIndexOf('.') + 1) == "properties" && modifiedRecords.filter(function(record) {
+        return record.path == file;
+      }).length <= 0;
+  }
+
+  run_command('git ls-tree --name-status -r HEAD', cwd, function(err, output) {
+    /* istanbul ignore if */
+    if (err) return cb(err);
+
+    var records = [];
+    var files = output.split('\n');
+    files.forEach(function(file) {
+      if(is_properties_not_modified(modifiedRecords, file)) {
```

Comment:
Would it be better to name the function is_properties_modified? just thinking of double negatives here :)